changelog.md

```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [Unreleased]
+
+### Added
+
+- Migration guides.
+- Rework on all events.
+- Store of event logs.
+
+### Changed
+
+- Remove old events management in `DefaultDrawer`.
+
+### Fixed
+
+- Prevent dependencies auto-updating in package-lock with `=` suffix.
```

Comment:
- Prevent auto-updating of dependencies in package-lock with `=` suffix.

Reply:
done